agent.cpp

```diff
@@ -430,6 +459,7 @@ void Agent::found_wumpus() {
 * Causes our agent to traverse the matrix and look for the gold.
 */
 void Agent::traverse_matrix() {
+  sleep(2);
```

Comment:
I hate magic numbers a lot.  Can we make this into a variable that we can control in the UI?  Or a define?